### DISCUSSION TITLE
Enhancing Projects Privacy: Selective Public and Private View Options

### DISCUSSION THREAD

**Discussion by sedghi** [community/community#58873] [Projects and Issues]
### Select Topic Area

Product Feedback

### Feature Area

Projects

### Body

It would be great if we can selectively choose which views in the Projects be public and which be private, currently it is only possible to set the WHOLE project as public or private.

![image](https://github.com/community/community/assets/7490180/7d2dda86-f519-47c4-bb56-d1370da41c45)

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by FAMOUSCLEAN** [comment]
I'm not a pro but I believe this is done to keep honest people honest one can always keep private notes and email others if there is something you choose not to share publicly.

**Reply by sedghi** [reply]
hmmm interesting take, but it is very common to have info such as roadmap, what is included in next release as Public, and other things such as estimates, costs etc. as Private. I mean if Github Projects is aiming to replace other project management apps...
